sources/packages/backend/libs/integrations/src/sfas-integration/sfas-integration.processing.service.ts

```diff
@@ -142,6 +142,11 @@ export class SFASIntegrationProcessingService {
       // Delete the file only if it was processed with success.
       if (result.success) {
         try {
+          await this.sfasIndividualService.updateStudentId();
+          await this.sfasIndividualService.updateBCSLDisbursementOveraward();
```

Comment:
I believe that at least the updates and inserts can be grouped and executed in parallel. Thoughts?

Reply:
That would be perfect.

Reply:
Yes, the 2 updates and 2 inserts to be executed in parallel works.